app/models/card.rb

```diff
@@ -2,6 +2,15 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, presence: true, uniqueness: true
   validate :fields_are_not_equal
   before_create :set_review_date
+  scope :actual, -> { where('review_date <= ?', Time.now).order("RANDOM()") }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.